docs/pages/upgrade-v3/oauth.md

```diff
@@ -96,7 +96,7 @@ if (existingAccount) {
 	});
 }
 
-// v2 IDs have length of 15
```

Comment:
I don't think this needs to be fixed. It's to show how to generate IDs that work with v2 projects

Reply:
ah ok. It's 15 [here for v3 ](https://v3.lucia-auth.com/basics/users) and we didn't have to worry about it for v2, so I was confused. thanks